nx_parallel/external.py

```diff
@@ -0,0 +1,88 @@
+import joblib
+from nx_parallel.misc import optional_package
+from typing import Optional
+
+SUPPORTED_BACKENDS = [
+    "multiprocessing",
+    "dask",
+    "ray",
+    "loky",
+    "threading",
+    "ipyparallel",
+]
+
+class Backend:
```

Comment:
Networkx implements a config context manager class which has been made compatible with nx-parallel in PR #75 . Also `joblib.parallel_config` can also be used to configure nx-parallel after PR #75 would be merged. Thanks!